models/user.js

```diff
@@ -0,0 +1,16 @@
+const md5 = require('md5');
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+    firstname: String,
+    email: String,
+    lastname: String,
+    password: String,
+    user_profiles: [{
```

Comment:
why user profile info needs to store in user Schema, duplicate work is here

Reply:
removed second ref which was present in user_profile collection